packages/flagship/ios/FLAGSHIP.xcodeproj/project.pbxproj

```diff
@@ -342,12 +365,18 @@
 				INFOPLIST_FILE = FLAGSHIP/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
```

Comment:
I had to change this line to:
`LD_RUNPATH_SEARCH_PATHS = "/usr/lib/swift $(inherited) @executable_path/Frameworks";`
to get it to run.

Reply:
Thanks for taking care of that. I was able to run iOS without it, but adding it doesn't seem to harm anything.